tamr_unify_client/models/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,75 @@
+from tamr_unify_client.models.base_collection import BaseCollection
+from tamr_unify_client.models.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(BaseCollection):
+    """Collection of :class'tamr_unify_client.models.project.attribute_mapping.AttributeMapping'
```

Comment:
Needs to be :class`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping` for docs